fidelius-ui/src/app/edit/edit.component.html

```diff
@@ -108,7 +158,7 @@
                              layout="column"
                              #secretRadioElement
                              name="secretTypeRadio"
-                             required
+                             required=false
```

Comment:
Not sure why are we making secretType as false? As of now, we have this as required.

Reply:
This needs to be false to allow edits to just the metadata. There was additional logic added to send edits just for secrets, metadata, or both